index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en" ng-app>
+<head>
+  <meta charset="UTF-8">
+  <title></title>
+</head>
+<body>
+
+  {{"hello" + " " + "world"}}
```

Comment:
This is a practice comment, to show how to make comments when doing a code review.